types/node/fs.d.ts

```diff
@@ -260,6 +260,29 @@ declare module 'fs' {
          */
         readSync(): Dirent | null;
     }
+    /**
+     * Class: fs.StatWatcher
+     * @since v14.3.0, v12.20.0
+     * Extends `EventEmitter`
+     * A successful call to {@link watchFile} method will return a new fs.StatWatcher object.
+     */
+    export class StatWatcher extends EventEmitter {
```

Comment:
It looks like this class does not exist, please make it an interface instead.

Reply:
To clarify:

Reply:
You're right, good catch!